provider/osio/osio.go

```diff
@@ -19,6 +20,12 @@ const (
 	authorization = "Authorization"
 )
 
+// list of cluster hard coded to be filtered out when choosing default backend
+var filterDefaultBackends = map[string]bool{
+	"https://api.starter-us-east-1a.openshift.com": true,
+	"https://api.starter-us-east-1b.openshift.com": true,
+}
+
```

Comment:
Here array would also be fine to hold list of URLs but I have some code in following PR [link](https://github.com/fabric8-services/fabric8-oso-proxy/pull/28) for which map would be better.